cmd/transcriber/call/transcriber.go

```diff
@@ -75,6 +84,9 @@ func (t *Transcriber) Start(ctx context.Context) error {
 	})
 	t.client.On(client.RTCTrackEvent, t.handleTrack)
 	t.client.On(client.CloseEvent, func(_ any) error {
+		go t.captionDoneOnce.Do(func() {
+			close(t.captionDoneCh)
+		})
```

Comment:
Any reason for not calling this inside the existing `t.done()` method?

Reply:
Huh, I think that might have changed -- before it wasn't working (causing a deadlock), but now it does. Thanks.